packages/web-components/fast-components/src/design-system-provider/index.ts

```diff
@@ -227,6 +231,38 @@ export class DesignSystemProvider extends FoundationElement {
     @designToken(density)
     public density: number;
 
+    /**
+     *
+     * The density offset, used with designUnit to calculate height and spacing.
+     *
+     * @remarks
+     * HTML attribute: density-component-vertical-adjustment-units
+     *
+     * CSS custom property: --density-component-vertical-adjustment-units
+     */
+    @attr({
```

Comment:
These two attributes were added for example and for testing. To follow the pattern all other density attributes would be added here as well.